packages/vue/src/components/atoms/SfColor/SfColor.html

```diff
@@ -0,0 +1,6 @@
+<button v-on="$listeners" class="sf-color" :class="{ 'sf-color--active' : isActive }" :aria-label="ariaLabel">
```

Comment:
@pspaczek Imo it wasnt a good idea as it would require lots of removing unnecessary styling, also for similar cases like SfArrow we use native` <button>`

Reply:
@aniamusial 👍ok